client/common/latexify.ts

```diff
@@ -401,12 +480,12 @@ function formatSuperTitle(row: IRow): Array<string> {
         if (row.cells[i]) {
             if (row.cells[i].content) {
                 if (empty > 0) {
-                    cells.push(`\\multicolumn{${ empty }}{c}{~}`);
```

Comment:
the use of template literals would have been fine here ... but whichever you prefer